kolibri/plugins/coach/assets/src/reportConstants.js

```diff
@@ -32,10 +33,13 @@ const SortOrders = {
   NONE: 'none',
 };
 
+const RECENCY_THREHOLD_IN_DAYS = 7;
```

Comment:
Typo!

Also, I get the sense that this might be more appropriate in the state, rather than a constant. Thoughts?